shared/src/main/java/com/couchbase/lite/ReadOnlyDocument.java

```diff
@@ -161,26 +164,23 @@ else if (c == '-')
         return 0;
     }
 
-    /*package*/ CBLC4Doc getC4doc() {
+    CBLC4Doc getC4doc() {
         return c4doc;
     }
 
     // Reads the document from the db into a new C4Document and returns it, w/o affecting my state.
-    /*package*/ com.couchbase.litecore.Document readC4Doc(boolean mustExist) throws CouchbaseLiteException, IllegalStateException {
-        try {
-            return database.getC4Database().getDocument(getId(), mustExist);
-        } catch (LiteCoreException e) {
-            throw LiteCoreBridge.convertException(e);
-        }
+    com.couchbase.litecore.Document readC4Doc(boolean mustExist)
+            throws LiteCoreException {
+        return database.getC4Database().getDocument(getId(), mustExist);
     }
 
-    /*package*/ ConflictResolver effectiveConflictResolver() {
+    ConflictResolver effectiveConflictResolver() {
         return getDatabase().getConflictResolver() != null ?
                 getDatabase().getConflictResolver() :
                 new DefaultConflictResolver();
     }
 
-    /* package */ boolean selectConflictingRevision() {
+    boolean selectConflictingRevision() throws CouchbaseLiteException {
```

Comment:
Is CouchbaseLiteException thrown from anything in this method? 
Or it should return void but throw CouchbaseLiteException instead?